sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -124,9 +152,12 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
             if (getCustomDtResponse.GetRawResponse().Status == (int)HttpStatusCode.OK)
             {
                 CustomDigitalTwin customDt = JsonSerializer.Deserialize<CustomDigitalTwin>(getCustomDtResponse.Value);
-                Console.WriteLine($"Retrieved and deserialized digital twin {customDt.Id} with ETag {customDt.ETag} " +
-                    $"and Prop1: '{customDt.Prop1}', Prop2: {customDt.Prop2}," +
-                    $"ComponentProp1: '{customDt.Component1.ComponentProp1}', ComponentProp2: {customDt.Component1.ComponentProp2}");
+                Console.WriteLine($"Retrieved and deserialized digital twin {customDt.Id}:\n\t" +
+                    $"ETag: {customDt.ETag}\n\t" +
```

Comment:
We should add this for Snippet:DigitalTwinsSampleGetBasicDigitalTwin in L107 as well